nupic/research/frameworks/vernon/mixins/prune_low_snr.py

```diff
@@ -102,11 +102,23 @@ def pre_epoch(self):
                     (self.current_timestep, result)
                 )
 
+    def run_epoch(self):
+        results = super().run_epoch()
+        vdrop_data = self.model.module.vdrop_data
+        total_prunable_parameters = vdrop_data.z_mask.size(0)
+        remaining_nonzero_parameters = vdrop_data.z_mask.double().sum().item()
+        model_sparsity = remaining_nonzero_parameters/total_prunable_parameters
+        results["total_prunable_parameters"] = total_prunable_parameters
+        results["remaining_nonzero_parameters"] = remaining_nonzero_parameters
+        results["model_sparsity"] = model_sparsity
```

Comment:
We've tried to get out of the habit of calling this a "sparsity". It's a "density". And we always want to be clear about whether we're talking about activation sparsity or weight sparsity, it should probably be "weight_density" or "model_weight_density".